command/cp.go

```diff
@@ -131,6 +131,10 @@ var copyCommandFlags = []cli.Flag{
 		Name:  "acl",
 		Usage: "set acl for target: defines granted accesses and their types on different accounts/groups",
 	},
+	&cli.BoolFlag{
+		Name:  "force-glacier-transfer",
+		Usage: "omit checks for glacier objects",
```

Comment:
A sentence stating that it would try to issue copy for files with GLACIER storage class whether they're restored or not would be better.

Reply:
changed to `try to copy files which is restored from GLACIER storage class`.

Reply:
It kinda sounds like the option only works on restored objects. With force option, if ile is not restored, S3 would still raise an error;
```
InvalidObjectState: The operation is not valid for the object's storage class status
```
I think we should indicate we do not check restore status and issue a copy request anyways.